velox/functions/prestosql/tests/BinaryFunctionsTest.cpp

```diff
@@ -697,4 +699,57 @@ TEST_F(BinaryFunctionsTest, toIEEE754Bits32) {
       hexToDec("FF7FFFFF"),
       toIEEE754Bits32(std::numeric_limits<float>::lowest()));
 }
+
+TEST_F(BinaryFunctionsTest, ChecksPadding) {
```

Comment:
These tests are alright. But BinaryFunctions is a test of prestosql binary functions. Might be worth moving this test logic into common/encode/tests folder.

Reply:
Moved the test case as per velox standard